.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Your github action
+
+on:
+  push:
+    branches:
+      # NOTE: You may want to limit the trigger branch to be "main" or "master" etc.
+      - '*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: install
+
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: docs-build
+
+      - name: Deploy to GitHub Pages
+        if: success()
+        uses: crazy-max/ghaction-github-pages@v2
```

Comment:
how'd you land on this one?  [This one](https://github.com/peaceiris/actions-gh-pages) for instance has 10x as many stars, so maybe let's use that one?